src/test/kotlin/fi/metatavu/timebank/api/test/functional/data/TestDateUtils.kt

```diff
@@ -0,0 +1,93 @@
+package fi.metatavu.timebank.api.test.functional.data
+
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+/**
+ * Class for utility methods for Dates used in tests
+ */
+class TestDateUtils {
+
+    companion object {
+
+        /**
+         * Gets given date as OffsetDateTime and removes time offset
+         *
+         * @return String OffsetDateTime without time offset
+         */
+        fun getODT(dateTime: LocalDateTime): String {
```

Comment:
LocalDateTime is time zone agnostic, so this seems a bit odd method. Why you don't use OffsetDateTime at Z and serialize that?